src/modules/quoiFeur/quoiFeur.helpers.ts

```diff
@@ -35,11 +35,16 @@ export const reactOnEndWithQuoi = async (message: Message) => {
   const probability = 1 / 6;
 
   if (Math.random() <= probability) {
-    await reactWithCoubeh(message);
-    await message.member?.timeout(
-      ONE_MINUTE * 5,
-      `${message.member.displayName} have the cramptés`,
-    );
+    try {
+      await reactWithCoubeh(message);
+    } catch (error) {
+      console.log(error)
```

Comment:
Could you check if the reason of the throw is the disappearance of the message ?
We don't want to silence every other errors possible.
For any other errors just re-throw them.

You might need to change the fact that you timeout in the `finally` for it to work like we want.